frontend/task/task.scss

```diff
@@ -309,6 +309,108 @@ body {
       }
     }
   }
+
+  .task-recorder-controls {
```

Comment:
I think it would be better to add some padding on the left side (8px for instance?), to avoid having the record button right in the corner. What do you think?

Reply:
I'm not convinced since the buttons are now bigger but I have added 2px of padding-left :smile:

Reply:
You're right, it's probably not that useful now that the buttons are bigger!